src/lib/semantic-versioning.ts

```diff
@@ -0,0 +1,39 @@
+import { isArray } from "lodash"
```

Comment:
#superminor - most files in the `lib` directory follow conventional `camelCase` file naming

Reply:
Fixed in 439e01d